cros_gralloc/gralloc4/CrosGralloc4Mapper.cc

```diff
@@ -150,7 +150,14 @@ Return<Error> CrosGralloc4Mapper::validateBufferSize(void* rawHandle,
     }
 
     PixelFormat crosHandleFormat = static_cast<PixelFormat>(crosHandle->droid_format);
-    if (descriptor.format != crosHandleFormat &&
+
+    uint32_t descriptor_drmFormat = 0, handle_drmFormat = 0;
+    handle_drmFormat = cros_gralloc_convert_format(static_cast<int32_t>(crosHandleFormat));
+    if (convertToDrmFormat(descriptor.format, &descriptor_drmFormat)) {
+        descriptor_drmFormat = cros_gralloc_convert_format(static_cast<int32_t>(descriptor.format));
+    }
+
+    if (descriptor_drmFormat != handle_drmFormat &&
```

Comment:
'convertToDrmFormat' is a little duplicated with 'cros_gralloc_convert_format'. Also 'PixelFormat::YCBCR_P010' is included in 'convertToDrmFormat'. Should we add 'i915_private_convert_format' into 'convertToDrmFormat', and replace 'cros_gralloc_convert_format' with 'convertToDrmFormat' in the whole gralloc4?  @feijiang1 @renchenglei @yhe39